pubsub/api/Pubsub.Samples/CreateAvroSchema.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_create_avro_schema]
+
+using Google.Cloud.PubSub.V1;
+using Grpc.Core;
+using System;
+
+public class CreateAvroSchemaSample
+{
+    public Schema CreateAvroSchema(string projectId, string schemaId, string pathToDefinition)
+    {
+        SchemaServiceClient schemaService = SchemaServiceClient.Create();
+        var schemaName = SchemaName.FromProjectSchema(projectId, schemaId);
+        string schemaDefinition = System.IO.File.ReadAllText(pathToDefinition);
```

Comment:
Unless if needed for disambiguation, we preffer `using System.IO` and then `File.Read...` here.